modules/tools/rode/tfsec-collector-values.yaml.tpl

```diff
@@ -0,0 +1,19 @@
+rode:
+  host: rode.${namespace}.svc.cluster.local:50051
```

Comment:
They should be in the same namespace, right? idk which is better practice, but I know the UI just uses `rode` to get to the rode service.

Reply:
Probably doesn't make a different, but I thought using the FQDN saved some extra DNS lookups. Couldn't find a source, but tried on a pod:

```shell
/usr/src/app # nslookup rode.rode-demo.svc.cluster.local
Server:		10.43.0.10
Address:	10.43.0.10:53

Name:	rode.rode-demo.svc.cluster.local
Address: 10.43.92.233

/usr/src/app # nslookup rode
Server:		10.43.0.10
Address:	10.43.0.10:53

Name:	rode.rode-demo.svc.cluster.local
Address: 10.43.92.233

** server can't find rode.svc.cluster.local: NXDOMAIN

** server can't find rode.cluster.local: NXDOMAIN

** server can't find rode.svc.cluster.local: NXDOMAIN

** server can't find rode.cluster.local: NXDOMAIN
```